internal/models/token_options.go

```diff
@@ -0,0 +1,43 @@
+// Package models contains models needed in client and server
+package models
```

Comment:
This probably not belong to the /internal package, because it has nothing to do with TPM.
This might belong to the verifier or the launcher package